Mathlib/Analysis/InnerProductSpace/Adjoint.lean

```diff
@@ -288,20 +288,41 @@ theorem _root_.LinearMap.IsSymmetric.isSelfAdjoint {A : E →L[𝕜] E}
     (hA : (A : E →ₗ[𝕜] E).IsSymmetric) : IsSelfAdjoint A := by
   rwa [← ContinuousLinearMap.isSelfAdjoint_iff_isSymmetric] at hA
 
+end IsSelfAdjoint
+
 /-- The orthogonal projection is self-adjoint. -/
-theorem _root_.orthogonalProjection_isSelfAdjoint (U : Submodule 𝕜 E) [CompleteSpace U] :
-    IsSelfAdjoint (U.subtypeL ∘L U.orthogonalProjection) :=
+@[simp]
+theorem _root_.isSelfAdjoint_starProjection [CompleteSpace E]
+    (U : Submodule 𝕜 E) [U.HasOrthogonalProjection] :
+    IsSelfAdjoint U.starProjection :=
   U.orthogonalProjection_isSymmetric.isSelfAdjoint
 
-theorem conj_orthogonalProjection {T : E →L[𝕜] E} (hT : IsSelfAdjoint T) (U : Submodule 𝕜 E)
-    [CompleteSpace U] :
-    IsSelfAdjoint
-      (U.subtypeL ∘L U.orthogonalProjection ∘L T ∘L U.subtypeL ∘L U.orthogonalProjection) := by
-  rw [← ContinuousLinearMap.comp_assoc]
-  nth_rw 1 [← (orthogonalProjection_isSelfAdjoint U).adjoint_eq]
-  exact hT.adjoint_conj _
```

Comment:
I realize you've likely deleted this because I said to do so. But probably we should instead do:
```lean
theorem conj_starProjection {T : E →L[𝕜] E} (hT : IsSelfAdjoint T)
    (U : Submodule 𝕜 E) [U.HasOrthogonalProjection] :
    IsSelfAdjoint (U.starProjection ∘L T ∘L U.starProjection) :=
  show IsSelfAdjoint (U.starProjection * T * U.starProjection)
    from hT.conjugate_self <| isSelfAdjoint_starProjection U

@[deprecated (since := "2025-07-05")] alias conj_orthogonalProjection := conj_starProjection

end IsSelfAdjoint
```
after the deprecation of `orthogonalProjection_isSelfAdjoint`. And the end of the section should be moved down here.